api/net/Dockerfile

```diff
@@ -34,13 +34,15 @@ RUN chmod +x /app/entrypoint.sh
 RUN mkdir /data
 VOLUME /data
 RUN chgrp -R 0 /data && \
-    chmod -R g=u /data
+    chmod -R g=u /data && \
+    chown -R 1001 /data
```

Comment:
Make the local api build similar to the one running in Openshift.  This was causing permission issues with folders/files created by the root user.